source/_posts/2020-05-20-release-110.markdown

```diff
@@ -0,0 +1,1477 @@
+---
+layout: post
+title: "0.110: Beta release notes"
+description: "0.110: Beta release notes"
+date: 2020-05-12 00:00:00
+date_formatted: "May 20, 2020"
+author: Franck Nijhof
+author_twitter: frenck
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-05-0.110/social.png
+---
+
+<a href='/integrations/#version/0.110'><img src='/images/blog/2020-05-0.110/components.png' style='border: 0;box-shadow: none;'></a>
+
+Let's keep the introduction for the Home Assistant Core 0.110 release short,
+as this release is stuffed! Starting with 12! new awesome integrations added
+this release!
+
+## Icons
+
+In Home Assistant Core [0.109](/blog/2020/04/29/release-109/#weight-loss-for-the-frontend),
+we made the frontend lighter and faster, this release takes it a step further.
+
+The way icons are loaded is updated, with the ever-growing Material Icons set,
+it was necessary to update the way we handle icons to make sure our application
+stays performing.
+
+The Material Icons are now split in chunks, so the frontend does not have to
+load all the icons if you just need one; besides that, we no longer store the
+icons in the DOM but in a database.
+
+This saves a lot of memory and thus makes the Home Assistant frontend,
+even more, leaner and faster!
+
+## Integrations grouping, searching & custom logos
+
+[@timmo001] added the possibility to search your integrations, so you can find
+what you are looking for quickly.
+
+We also grouped the entries by integration now, this means we no longer show
+multiple cards for the same integration, but show a list of the names if there
+are multiple entries.
+
+If you click the entry, it will show that entry in the card.
+This makes it easier to keep an overview of all your integrations.
+
+<p class='img'>
+<img src='/images/blog/2020-05-0.110/integrations.png' alt='Screenshot of the integrations page'></a>
+Screenshot of the integrations page.
+</p>
+
+Oh! We've also made the icons and logos available for custom integrations!
+
+<p class='img'>
+<img src='/images/blog/2020-05-0.110/custom_logo.png' alt='Screenshot of the HACS integration with its icon shown'></a>
+Screenshot of the HACS integration with its icon shown.
+</p>
+
+[@timmo001]: https://github.com/timmo001
```

Comment:
Looks out of place, should probably be at the bottom with the rest (not that anyone sees it)

Reply:
No, because the bottom part is generated and will be replaced on the final release. Hence links not provided by the bottom part are in-line.